src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -18,6 +18,7 @@ import {
   TAB_KEYCODE,
 } from '../../util/keycodes';
 
+// This component is deprecated and will be replaced by the Menu component
```

Comment:
sounds good. also another supporting fact for grouping things together since marking subcomponents as deprecated gets complicated if they are all separated out.